src/libstd/primitive_docs.rs

```diff
@@ -347,13 +330,31 @@ mod prim_pointer { }
 /// ```
 ///
 /// If the array has 32 or fewer elements (see above), you can also use the
-/// array reference's `IntoIterator` implementation:
+/// array reference's [`IntoIterator`] implementation:
```

Comment:
We generally try to keep the syntax I've put in here. So not sure if I should change it or not...